code/__DEFINES/reagents.dm

```diff
@@ -7,10 +7,11 @@
 
 #define REFILLABLE		(1<<2)	// Makes it possible to add reagents through any reagent container.
 #define DRAINABLE		(1<<3)	// Makes it possible to remove reagents through any reagent container.
+#define DUNKABLE		(1<<4)	// Allows items to be dunked into this container for transfering reagents. Used in conjunction with the dunkable component.
```

Comment:
Do we want a flag just for this? I feel like there'll be a lot of classes that need this flag but don't have it

Reply:
I thought it would be preferable to having a string of istypes like the original proc had, and instead allows us to check for just a single flag.

Reply:
Why not use IS_OPEN_CONTAINER or whatever it was called?

Reply:
Because dunking a donut in a plastic bottle is wild

Reply:
That valid

Reply:
have you SEEN the diameter of those g*torade bottles?

Reply:
That sounds like quitter talk.

Reply:
yeah you can get the neck of a wine bottle into those things